PyAlbany/tests/small/IO/CMakeLists.txt

```diff
@@ -0,0 +1,17 @@
+##*****************************************************************//
+##    Albany 3.0:  Copyright 2016 Sandia Corporation               //
+##    This Software is released under the BSD license detailed     //
+##    in the file "license.txt" in the top-level Albany directory  //
+##*****************************************************************//
+
+add_test(PyAlbany_IO "${PYTHON_EXECUTABLE}" "${CMAKE_CURRENT_SOURCE_DIR}/io_PyAlbany.py")
```

Comment:
I think it is better if you copy the test files (e.g. io_PyAlbany.py) in the build folder, as done in the other tests. In  that way you don't have to link the source directory. Same for the other tests

Reply:
I updated the tests; the files are now copied during the configuration.